src/main/java/racingcar/io/input/InputManager.java

```diff
@@ -0,0 +1,55 @@
+package racingcar.io.input;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+import racingcar.global.CharacterSymbol;
+import racingcar.io.utils.StringLiteralValidator;
+
+public class InputManager {
+
+    private static final String CAR_NAME_DELIMITER = CharacterSymbol.COMMA.getLiteral();
+
+    private final Reader reader;
+
+    public InputManager(Reader reader) {
+        this.reader = reader;
+    }
+
+    public List<String> readCarNames() {
+        String input = reader.readLine();
+
+        List<String> delimited = parseByDelimiter(input);
+
+        validateBlankCharacterNotIncluded(delimited);
+
+        return delimited;
+    }
+
+    public long readMovingCount() {
+        String input = reader.readLine();
+
+        StringLiteralValidator.validateIsPositiveInteger(input);
+
+        return Long.parseLong(input);
+    }
+
+    private List<String> parseByDelimiter(String input) {
+        return Arrays.stream(input.split(CAR_NAME_DELIMITER))
+                .collect(Collectors.toList());
```

Comment:
```suggestion
               .toList();
```
를 쓰는 방법도 있습니다! 내부에서 unmodifiableList도 함께해주더라고요!
```java
    default List<T> toList() {
        return (List<T>) Collections.unmodifiableList(new ArrayList<>(Arrays.asList(this.toArray())));
    }
```

Reply:
오 꿀팁이네요! 배워갑니다